beta1/integrated-build.json

```diff
@@ -93,8 +93,8 @@
         "strategy": {
           "type": "STI",
           "stiStrategy": {
-            "image": "openshift/ruby-20-centos",
-            "scripts": "https://raw.githubusercontent.com/openshift/ruby-20-centos/master/.sti/bin"
+            "image": "openshift/ruby-20-centos7",
+            "scripts": "https://raw.githubusercontent.com/openshift/sti-ruby/master/2.0/.sti/bin"
```

Comment:
The `scripts` defined here are just an option you can but not have to use. By default all those images both old ones and new ones have internal var set ('STI_SCRIPTS_URL') which points to that location. What I'd propose at this point is to remove that scripts variable from here, as this makes the buildConfig less readable, it's an optional field.